pom.xml

```diff
@@ -35,7 +35,7 @@
 
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>publish-over</artifactId>
-    <packaging>jar</packaging>
+    <packaging>hpi</packaging>
     <name>Publish Over</name>
```

Comment:
Name needs to be changed. "Utilities for 'Publish Over' plugins" or so?

Reply:
Maybe. Not a priority for me.